.github/workflows/main.yml

```diff
@@ -92,6 +92,10 @@ jobs:
             - api: '31'
               abi: 'x86_64' # x86 ABI is not yet available for API 31
               tag: 'google_apis'
+
+            - api: '32'
+              abi: 'x86_64'
```

Comment:
Do you use the x86_64 ABI as it is not made available by Google ? (yet?) 
Could you please check if x86 is available on API 31 ? If not then maybe we can assume that on API >= 31, then only x86_64 is available and would could move/rewrite the comment above ?

Reply:
Yes, x86 is not available for API 31. The same is true for API 32 and 33. I guess we can assume Google won't be releasing x86 emulators. I will remove comment then

Reply:
![image](https://user-images.githubusercontent.com/110480725/190978320-083d9931-f464-4980-9195-ad5152ef7878.png)

Reply:
It is from [here](https://developer.android.com/about/versions/12/get)